src/main/webapp/templates/questionChoiceItemView.html

```diff
@@ -1,26 +1,20 @@
 
```

Comment:
Not sure why this file needed to change??

Reply:
this is where I tried to revert the changes with another commit